pkg/util/workerqueue/workerqueue_test.go

```diff
@@ -194,3 +196,11 @@ func TestWorkerQueueEnqueueAfter(t *testing.T) {
 		assert.Fail(t, "should have got a queue'd message by now")
 	}
 }
+
+func TestIsDebugError(t *testing.T) {
```

Comment:
Add a test calling `NewDebugError` with nil as an argument.

Reply:
ah - that is a very good idea!